modules/iscsi/udiskslinuxmanageriscsiinitiator.c

```diff
@@ -282,86 +283,105 @@ handle_get_firmware_initiator_name (UDisksManagerISCSIInitiator *object,
   return TRUE;
 }
 
+
+static gchar* _get_initiator_name (GError **error)
+{
+  gchar *initiator_name = NULL;
+  gchar *initiator_contents = NULL;
+  gsize  initiator_size = 0;
```

Comment:
Unused variable :-p

(not sure whether compiler complains about unused variable when initial value is assigned like this...)

Reply:
Oops, fixed.